src/map/script.cpp

```diff
@@ -5566,33 +5566,46 @@ BUILDIN_FUNC(return)
 	return SCRIPT_CMD_SUCCESS;
 }
 
-/// Returns a random number from 0 to <range>-1.
-/// Or returns a random number from <min> to <max>.
-/// If <min> is greater than <max>, their numbers are switched.
-/// rand(<range>) -> <int>
-/// rand(<min>,<max>) -> <int>
+/// Returns a random number.
+/// rand(<range>) -> <int64> in the mathematical range [0, <range>[
```

Comment:
```suggestion
/// rand(<range>) -> <int64> in the mathematical range [0, <range>]
```

Reply:
Its exclusive <range>.
If we want to write it with inclusive definition we would have to put <range - 1>.

Reply:
Then it'd be `[0-range)`
Unless there's a regional difference